decidim-admin/app/controllers/decidim/admin/scopes_controller.rb

```diff
@@ -14,7 +14,7 @@ def index
                     parent_scope.children
                   else
                     collection.top_level
-                  end
+                  end .order "name->'#{I18n.locale}' ASC"
```

Comment:
:+1: for this, but the style looks weird to me. I would extract the old logic to a method or local variable, and append the ordering to it.